Plugins/JiraCommitHintPlugin/JiraCommitHintPlugin.cs

```diff
@@ -25,11 +27,14 @@ public class JiraCommitHintPlugin : GitPluginBase, IGitPluginForRepository
         private static readonly TranslationString EmptyQueryResultCaption = new("First Task Preview");
 
         private const string DefaultFormat = "{Key} {Summary}";
+        private const string BASIC = "Basic";
+        private const string JWT = "JWT";
```

Comment:
Please use sentence case for consts too.

```suggestion
        private const string AuthTypeBasic = "Basic";
        private const string AuthTypeJwt = "JWT";
```